bitcoin/src/psbt/raw.rs

```diff
@@ -16,15 +16,14 @@ use crate::consensus::encode::{
 use crate::prelude::{DisplayHex, Vec};
 use crate::psbt::Error;
 
-/// A PSBT key in its raw byte form.
+/// A PSBT key in its raw byte form, where `<key> := <keylen> <keytype> <keydata>`
```

Comment:
The period should be kept and I'd prefer to put it in a separate paragraph.

Reply:
> put it in a separate paragraph

Yes that is probably the best.
Maybe the autoformat bot PR could break into separates line what is inside `.

Reply:
IMO this is largely subjective. Even LLM would have trouble with this.

Reply:
Will add as a separate paragraph